pkg/installer/custominstallconfig_test.go

```diff
@@ -230,6 +255,8 @@ func mockClientCalls(client *azuremock.MockAPI) {
 			ID:       to.StringPtr("test-resource-group"),
 			Location: to.StringPtr("centralus"),
 		}, nil)
+	client.EXPECT().ListResourceIDsByGroup(gomock.Any(), "test-resource-group").
```

Comment:
This call only happens when you forget to add `-tags aro`. That's also why it's failing in CI.